syncapi/consumers/roomserver.go

```diff
@@ -327,12 +340,13 @@ func (s *OutputRoomEventConsumer) onRetirePeek(
 		}).Panicf("roomserver output log: write peek failure")
 		return nil
 	}
-	// tell the notifier about the new peek so it knows to wake up new devices
-	s.notifier.OnRetirePeek(msg.RoomID, msg.UserID, msg.DeviceID)
```

Comment:
I didn't see where this got moved to?

Reply:
This was a mistake, have put it back, but added the position into the parameters too, since we still need to update the PDU position for peeks for now.